src/ipc-handlers.ts

```diff
@@ -676,8 +682,25 @@ export async function exportSite(
 			const parentWindow = BrowserWindow.fromWebContents( event.sender );
 			sendIpcEventToRendererWithWindow( parentWindow, 'on-export', data, siteId );
 		};
-		return await exportBackup( options, onEvent );
+
+		const result = await exportBackup( options, onEvent );
+
+		if ( result ) {
+			bumpStat(
+				StatsGroup.STUDIO_EXPORT,
+				options.includes.database &&
+					! options.includes.uploads &&
+					! options.includes.plugins &&
+					! options.includes.themes &&
+					! options.includes.muPlugins
+					? StatsMetric.DATABASE_ONLY
+					: StatsMetric.FULL_SITE
+			);
```

Comment:
```suggestion
			const isDatabaseOnly =
				options.includes.database &&
				! options.includes.uploads &&
				! options.includes.plugins &&
				! options.includes.themes &&
				! options.includes.muPlugins;

			bumpStat(
				StatsGroup.STUDIO_EXPORT,
				isDatabaseOnly ? StatsMetric.DATABASE_ONLY : StatsMetric.FULL_SITE
			);
```
Nit, but this would read better if we assigned the condition to a variable.